test/bot_test.py

```diff
@@ -100,6 +103,7 @@ def join_channel(self, nick, channel):
             expected_calls.append((nick, 'private', 0, 'Joined channel %s' % channel, 'privmsg'))
         self.assertEqual(calls, expected_calls)
         self.log_mock.reset_mock()
+        self.raw_mock.reset_mock()
```

Comment:
This really shouldn't go here, the only reason log_mock is reset here is because the send_msg call affects it.